CHANGELOG.md

```diff
@@ -5,8 +5,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## 0.5.2
```

Comment:
@jadie1 quick fix here, could you create a new `Unreleased` section (rather than adding these changes under `0.5.2` which has already been tagged/released)

Reply:
Oops, yes I'll fix this